com.unity.formats.alembic/Third Party Notices.md

```diff
@@ -1,43 +1,24 @@
-This package contains third-party software components governed by the license(s) indicated below:
+This package contains third-party software components governed by the license(s) indicated below:  
 ---------
 
-# OpenEXR
+Component Name: OpenEXR  
+License Type: Modified BSD License  
+Copyright © 2006-2019 Series of LF Projects, LLC  
+https://www.openexr.com  
 
-Copyright (c) 2006-17, Industrial Light & Magic, a division of Lucasfilm
-Entertainment Company Ltd.  Portions contributed and copyright held by
-others as indicated.  All rights reserved.
+Copyright (c) 2006-2019 OpenEXR a Series of LF Projects, LLC. All rights reserved. 
```

Comment:
I forgot to point out this one: which Copyright line is the actual one to keep? There is another one above.